### ISSUE TITLE
Extended Properties Don't Work for CalendarEntry Objects

### ISSUE THREAD

**Issue by GoogleCodeExporter** [barsu8184/google-gdata#139]
```
What steps will reproduce the problem?
1. Create Calendar
2. Add Extended Property
3. Save

What is the expected output? What do you see instead?

Should create a calendar with an extended property, but the property is not
saved. After some debugging, it appears that the XML being generated is not
correct and does not contain the extended property.


What version of the product are you using? On what operating system?

1.1.4.1 on Windows Vista Business

Please provide any additional information below.

Please see attached Visual Studio Solution for an example.

```

Original issue reported on code.google.com by `nevada...@gmail.com` on 20 Jun 2008 at 5:52

Attachments:
- [ExtendedPropertyIssue.zip](https://storage.googleapis.com/google-code-attachments/google-gdata/issue-139/comment-0/ExtendedPropertyIssue.zip)

**Comment by GoogleCodeExporter** [comment]
```
Great sample, was easy to reproduce. I can verify that .NET creates the 
following entry to the server: 

﻿<?xml version="1.0" encoding="utf-8"?>
<entry xmlns="http://www.w3.org/2005/Atom" 
xmlns:gd="http://schemas.google.com/g/2005">
  <gCal:timezone value="America/Los_Angeles" xmlns:gCal="http://schemas.google.com/gCal/2005" />
  <gCal:hidden value="False" xmlns:gCal="http://schemas.google.com/gCal/2005" />
  <gd:extendedProperty name="group" value="admin" />
  <title type="text">ExtendedPropertyTest2</title>
</entry>

And that this entry comes back:

<?xml version='1.0' encoding='UTF-8'?><entry 
xmlns='http://www.w3.org/2005/Atom' 
xmlns:gCal='http://schemas.google.com/gCal/2005' 
xmlns:gd='http://schemas.google.com/g/2005'><id>http://www.google.com/calendar/f
eeds/default/ownc
alendars/full/nb6fop6j22q5rdn60j561ajpok%40group.calendar.google.com</id><publis
hed>2008-06-
23T07:37:02.141Z</published><updated>2008-06-23T07:37:02.000Z</updated><title 
type='text'>ExtendedPropertyTest2</title><link rel='alternate' 
type='application/atom+xml' 
href='http://www.google.com/calendar/feeds/nb6fop6j22q5rdn60j561ajpok%40group.ca
lendar.google.com/
private/full'/><link rel='http://schemas.google.com/acl/2007#accessControlList' 
type='application/atom+xml' 
href='http://www.google.com/calendar/feeds/nb6fop6j22q5rdn60j561ajpok%40group.ca
lendar.google.com/
acl/full'/><link rel='self' type='application/atom+xml' 
href='http://www.google.com/calendar/feeds/default/owncalendars/full/nb6fop6j22q
5rdn60j561ajpok%40g
roup.calendar.google.com'/><link rel='edit' type='application/atom+xml' 
href='http://www.google.com/calendar/feeds/default/owncalendars/full/nb6fop6j22q
5rdn60j561ajpok%40g
roup.calendar.google.com'/><author><name>ExtendedPropertyTest2</name></author><g
Cal:timezone 
value='America/Los_Angeles'/><gCal:hidden value='false'/><gCal:color 
value='#2952A3'/><gCal:selected 
value='false'/><gCal:accesslevel value='owner'/></entry>

which does not contain an extended property. So it is not a .NET issue. I am 
going to forward this to the 
calendar team. 
```

Original comment by `fman...@gmail.com` on 23 Jun 2008 at 7:39
- Changed state: **Reviewed**

**Comment by GoogleCodeExporter** [comment]
```
This is an issue in the calendar service:

http://code.google.com/p/gdata-issues/issues/detail?id=49


```

Original comment by `fman...@gmail.com` on 4 Jul 2008 at 12:29
- Changed state: **ByDesign**
